mono/metadata/gc-internal.h

```diff
@@ -371,6 +371,8 @@ extern gboolean log_finalizers;
 
 /* If set, do not run finalizers. */
 extern gboolean do_not_finalize;
+/* List of names of classes not to finalize. */
+extern gchar **do_not_finalize_class_names;
```

Comment:
Please prefix with `mono_` so we don't pollute the global namespace.

(I know we already do this here, but I've been trying to clean these up over time.)